common/components/BalanceSidebar/Promos.tsx

```diff
@@ -94,6 +58,15 @@ export default class Promos extends React.Component<{}, State> {
   }
 
   private navigateToPromo = (idx: number) => () => {
+    // stop rotating when user begins interacting with promos
+    clearInterval(this.timer);
```

Comment:
This is a nice touch I was going to suggest, glad you beat me to the punch!

Reply:
Thx 😄